docs/release/trg-0/trg-1-08.md

```diff
@@ -0,0 +1,31 @@
+---
+title: TRG 1.08 - Interfaces Documentation
```

Comment:
Should we change the title to "API documentation"? I think this TRG does not target UIs

Reply:
TRG 1.08 - Interface documentation (APIs)